app/src/main/res/values-ar/strings.xml

```diff
@@ -80,8 +80,9 @@
     <!-- FAQ -->
     <string name="faq_1_title">كيف يمكنني تغيير لون القطع؟</string>
     <string name="faq_1_text">في حال كان لديك عنصر واجهة مستخدم نشط واحد فقط، يمكنك إما إعادة إنشائه، أو استخدام الزر في إعدادات التطبيق لتخصيصه. إذا كان لديك العديد من الحاجيات النشطة، لن يكون الزر الموجود في إعدادات التطبيق متاحا. بما أن التطبيق يدعم تخصيص الألوان لكل عنصر واجهة مستخدم ، سيكون عليك إعادة إنشاء القطعة التي تريد تخصيصها.</string>
+    <string name="cant_load_files_over_internet">Simple Notes can not load files over the internet</string>
```

Comment:
move the string somewhere above please, not randomly at the bottom

Reply:
are you using Linux? I have a script for adding new strings so that they arent at the bottom, like if added by Android Studio

Reply:
> are you using Linux? I have a script for adding new strings so that they arent at the bottom, like if added by Android Studio

I'm on a MacBook. What's the script? I think it should work.

Reply:
has to be ran at `res` folder. It can glitch if the string contains special characters or so, but new lines can also be escaped with `\\n`

Reply:
> for example `find . -name "strings.xml" | xargs sed -i '/<string name="upgrade_calendar"/a \ <string name="delete_all_events_and_tasks">Delete all events and tasks</string>'`, it adds the "delete_all_events_and_tasks string after upgrade_calendar

Cool. I will save that somewhere. Thanks.